ThingAppraiser/Program.cs

```diff
@@ -7,10 +7,53 @@ class Program
     {
         static readonly string[] Movies = { "Allied", "Venom", "Sayonara no asa ni yakusoku no hana o kazaro" };
 
+        // Function for choosing input type
+        private static string[] getNames(string[] args)
```

Comment:
Need to separate all logic into new class which could work with files. Program class should consist like user defined class without encapsulated working logic. It would be difficult to modify and maintain such code in the future.

Reply:
For example, you can move this method to `Input` class. Also if there are no need to store some state you can add `static` keyword to your class and method definitions. It improves usability and readability.